graylog2-web-interface/src/pages/ShowMessagePage.tsx

```diff
@@ -64,17 +63,13 @@ const useStreams = () => {
   return { streams, allStreams };
 };
 
-const useMessage = (index: string, messageId: string) => {
-  const [message, setMessage] = useState<Message | undefined>();
+const useInputs = (sourceInputId, gl2SourceNode) => {
```

Comment:
Can we type the parameters while we are at it?

Reply:
I missed that, thanks.